HTML/styles.scss

```diff
@@ -0,0 +1,53 @@
+.level1 {
+    padding: 10px;
+    background-color: #f0f0f0;
+
+    .level2 {
+        padding: 10px;
+        background-color: #d0d0d0;
+
+        .level3 {
+            padding: 10p;
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/17#issuecomment-2208262726>#007008</a><span> - 07/04/2024, 07:11 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Typographical error in the padding property for '.level3' class. The value is set to "10p" instead of "10px".
</div>
<div id="fix">
<b>Fix:</b> Correct the padding value to "10px" for the '.level3' class.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -10,7 +10,7 @@
      .level3 {
 -        padding: 10p;
 +        padding: 10px;
         background-color: #b0b0b0;
```
</div>
</details>
</div>
<br>


</div>